### ISSUE TITLE
Can we add conformance results for k8s 1.6.11?

### ISSUE THREAD

**Issue by nkwangleiGIT** [cncf/k8s-conformance#74]
hi,
We're currently using k8s 1.6.11 as the base version of our product, is it valid to run against this version and get the test results?

As I can see the test results seems fine:
```
Ran 123 of 697 Specs in 2818.596 seconds
SUCCESS! -- 123 Passed | 0 Failed | 0 Pending | 574 Skipped PASS
```

**Comment by dankohn** [comment]
Conformance is only available for Kubernetes versions 1.7 and higher. So, no, those results are not valid.

**Comment by nkwangleiGIT** [comment]
ok, we'll run against a higher version, thanks!
